Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -33,7 +33,8 @@ extension XcodeGraph.Target {
         manifest: ProjectDescription.Target,
         generatorPaths: GeneratorPaths,
         externalDependencies: [String: [XcodeGraph.TargetDependency]],
-        fileSystem: FileSysteming
+        fileSystem: FileSysteming,
+        type: TargetType = .local
```

Comment:
```suggestion
        type: TargetType
```

I'd consider not adding a default, so we don't accidentally break this behavior again in the future.